packages/web/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "vite",
     "prebuild": "rm -rf ./dist",
-    "build": "tsc && vite build",
+    "build": "tsc && vite build && cp -R ./dist ../../srcbook/public",
```

Comment:
This copies `dist` into `public` such that it's `public/dist/{files}` but the express server is expecting `public/{files}`.

I think changing it to `cp -R ./dist/. ../../srcbook/public` will work